cosmosdb/sql/ps-account-firewall-create.ps1

```diff
@@ -0,0 +1,30 @@
+# Create an Azure Cosmos Account for Core (SQL) API
+$resourceGroupName = "myResourceGroup"
+$location = "West US"
+$accountName = "mycosmosaccount" # must be lower case.
+
+$locations = @(
+    @{ "locationName"="West US"; "failoverPriority"=0 },
+    @{ "locationName"="East US"; "failoverPriority"=1 }
+)
+
+# Add an IP range filter to the properties
+$ipRangeFilter = @{ "ipRangeFilter"="10.0.0.1" }
+
+$consistencyPolicy = @{
+    "defaultConsistencyLevel"="BoundedStaleness";
+    "maxIntervalInSeconds"=300;
+    "maxStalenessPrefix"=100000
+}
+
+$CosmosDBProperties = @{
+    "databaseAccountOfferType"="Standard";
+    "locations"=$locations;
+    "consistencyPolicy"=$consistencyPolicy;
+    "ipRangeFilter"= $ipRangeFilter;
+    "enableMultipleWriteLocations"="true"
+}
+
+New-AzResource -ResourceType "Microsoft.DocumentDb/databaseAccounts" `
```

Comment:
Same issue as `cosmosdb/sql/ps-account-create.ps1` - trivial example but with a lot of configuration.

Reply:
The configuration is the point here.